app/uk/gov/hmrc/apinotificationpull/controllers/CustomHeaderNames.scala

```diff
@@ -30,8 +30,11 @@ object CustomHeaderNames {
   
   val X_CLIENT_AUTHORIZATION_TOKEN = "x-client-authorization-token"
 
+  val customHeadersNames = Seq(X_CLIENT_ID_HEADER_NAME,ACCEPT_HEADER_VALUE,X_CLIENT_ID_HEADER_NAME,X_CLIENT_AUTHORIZATION_TOKEN)
+
+
   implicit def getHeadersFromHeaderCarrier[A](implicit hc: HeaderCarrier): SeqOfHeader = {
-    hc.headers
+    hc.headers(customHeadersNames)
```

Comment:
we might need to check with previous team if that's the definitive list of headers

Reply:
there is a list of all headers we can extract but not sure the use of them here

Reply:
On second thought this might be enough as it is only used in logger and as far we are concerned we're interested only in the ones mentioned above. Check if there are any tests to validate we still extract all the headers we need for debugging purposes